packages/ui/public/locales/en/translation.json

```
@@ -0,0 +1,80 @@
+  "Set up your Coong wallet now": "",
+  "Create New Wallet": "",
+  "Restore existing wallet": "",
+  "Coming soon": "",
+  "First, choose your wallet password": "",
+  "Welcome back": "",
+  "Your password will be used to encrypt accounts as well as unlock the wallet, make sure to pick a strong & easy-to-remember password": "Your password will be used to <strong>encrypt accounts as well as unlock the wallet</strong>, make sure to pick a <strong>strong & easy-to-remember</strong> password.",
+  "Unlock your wallet": "",
+  "Wallet password": "",
+  "Next": "",
+  "Next, confirm your wallet password": "",
+  "Confirm wallet password": "",
+  "Unlock": "",
+  "Write down the below 12 words and keep it in a safe place.": "",
+  "Back": "",
+  "Finish": "",
+  "I have backed up my recovery phrase": "",
+  "Finally, back up your secret recovery phrase": "",
+  "Accounts": "",
+  "Search by name": "",
+  "Address format": "",
+  "New Account": "",
+  "Create new account": "",
+  "Choose a name for your new account": "",
+  "New account name": "",
+  "Create": "",
+  "Cancel": "",
+  "My first account": "",
+  "Settings": "",
+  "Theme Mode": "",
+  "Language": "",
+  "Dark": "",
+  "System": "",
+  "Light": "",
+  "Lock the wallet": "",
+  "Open settings": "",
+  "Close settings": "",
+  "Reset wallet": "",
+  "Wallet Access Request": "",
+  "An application, self-identifying as request app name is requesting access your wallet from origin.": "An application, self-identifying as <strong>{{appName}}</strong> is requesting access your wallet from <strong>{{origin}}</strong>.",
+  "Setup your Coong wallet now to connect": "",
+  "Select the accounts you'd like to connect": "",
+  "Only connect if you trust the application": "",
+  "Connect": "",
+  "Set up wallet": "",
+  "Set up new wallet": "",
+  "Password's too short": "",
+  "Type again your chosen password to ensure you remember it.": "",
+  "Password does not match": "",
+  "Select all": "",
+  "Deselect all": "",
+  "No accounts selected": "",
+  "account(s) selected": "",
+  "Invalid Request": "",
+  "This page should not be open directly!": "",
+  "No accounts meet search query:": "",
+  "No accounts found in wallet": "",
+  "Create your first account now!": "",
+  "Click to copy address": "",
+  "Address copied!": "",
+  "UnknownRequest": "Unknown Request",
+  "UnknownRequestOrigin": "Unknown Request Origin",
+  "InvalidMessageFormat": "Invalid Message Format",
+  "KeypairNotFound": "Keypair Not Found",
+  "AccountNotFound": "Account Not Found",
+  "KeyringNotInitialized": "Keyring Not Initialized",
+  "PasswordIncorrect": "Password Incorrect",
+  "PasswordRequired": "Password Required",
+  "WalletLocked": "Wallet Locked",
+  "KeyringLocked": "Keyring Locked",
+  "AccountNameRequired": "Account Name Required",
+  "AccountNameUsed": "Account Name Used",
+  "InternalError": "Internal Error"
```

Please change the translation for Error Code as below.
```suggestion
  "UnknownRequest": "Unknown request",
  "UnknownRequestOrigin": "Unknown request origin",
  "InvalidMessageFormat": "Invalid message format",
  "KeypairNotFound": "Keypair not found",
  "AccountNotFound": "Account not found",
  "KeyringNotInitialized": "Keyring's not initialized",
  "PasswordIncorrect": "Password incorrect",
  "PasswordRequired": "Password required",
  "WalletLocked": "The wallet's locked, please unlock it first.",
  "KeyringLocked": "The keyring's locked, please unlock the wallet first.",
  "AccountNameRequired": "Account name's required",
  "AccountNameUsed": "Account name's already picked",
  "InternalError": "Internal error"
```